src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -7,22 +7,13 @@
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
-    private static final int DEFAULT_CAPACITY = 10;
+    private final int defaultCapacity;
     private T[] values;
     private int size;
 
     public ArrayList() {
-        this.values = (T[]) new Object[DEFAULT_CAPACITY];
-    }
-
-    public ArrayList(int initialCapacity) {
-        if (initialCapacity > 0) {
-            this.values = (T[]) new Object[initialCapacity];
-        } else if (initialCapacity == 0) {
-            this.values = (T[]) new Object[0];
-        } else {
-            throw new IllegalArgumentException("Wrong argument : " + initialCapacity);
-        }
+        defaultCapacity = 10;
```

Comment:
i was talking about size variable
default capcity should be constant as it was

Reply:
understood